policy/release/tasks.rego

```diff
@@ -64,18 +70,31 @@ deny[result] {
 	count(att.predicate.buildConfig.tasks) > 0
 
 	# collects names of tasks that are present in the attestation
-	attested_tasks := {t |
-		task_ref := att.predicate.buildConfig.tasks[_].ref
-		task_ref.kind == "Task"
-		bundle_ref := task_ref.bundle
+	attested_tasks_ary := {t |
+		task := att.predicate.buildConfig.tasks[_]
+		task.ref.kind == "Task"
+		bundle_ref := task.ref.bundle
 		bundles.is_acceptable(bundle_ref)
-		t := task_ref.name
+		t := _task_names(task)
```

Comment:
You can change this to `t := _task_names(task)[_]` to avoid the intermediate `attested_tasks_ary` variable.